Shopify/cli
# Remove print extension uniqueness validation

### WHY are these changes introduced?
Remove print extension validation which should live in Core.
issue [#1183](https://github.com/orgs/Shopify/projects/7965/views/29?pane=issue&itemId=65371800#:~:text=Validation%20to%20Core-,%231183,-Edit)


**NOTE**: This shouldn't be merged until core validation is merged in this [PR](https://github.com/Shopify/shopify/pull/523523)
### WHAT is this pull request doing?

Remove the validation as it is implemented in cor in this [PR](https://github.com/Shopify/shopify/pull/523523)

### How to test your changes?
- Create an app with two print extensions with the same target
- In the CLI project, checkout the branch `1183-move-cli-validation-to-core`
- In an spin instance, go to shopify shell and  checkout the branch `1183-move-cli-validation-to-core`
- Try to deploy the app using the local CLI branch, it should return an error mensioning the extension handles and the target

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
